doc/modules/model_evaluation.rst

```diff
@@ -473,6 +473,13 @@ given binary ``y_true`` and ``y_pred``:
       for C-class classification problem).
```

Comment:
Ah, looking at Guyon2015, I now see that "normalize" is not a multiplicative factor, but a chance correction. I would avoid "normalize" as this usually means scaling to ensure that a value falls in an expected range.

Looking at their code, I now see that their [their per-class accuracy is recall](https://github.com/ch-imad/AutoMl_Challenge/blob/20e708f51a12f31dc9e2d66c52d7df01453b9179/Starting_kit/scoring_program/libscores.py#L206), not rand or balanced accuracy.

All in all lots of this description is related to the input format setup in that challenge. All they are doing is computing average recall across classes, so this should be merged into the below definition. They then correct for chance, I think, to ensure that "at chance" scores 0 regardless of the number of classes in the task. This can be noted with citation to Guyon2015.

Again sorry I wasn't reviewing more critically in the first place.

Reply:
maybe "adjusted" instead of normalized would be better, but in either case, describing what is meant would be good.
Do you think giving formulas would be helpful? You just need to define recall and precision for each class and then writing down the three different formulas is pretty straight forward.
So the code you quote is not what they write in the paper http://www.causality.inf.ethz.ch/AutoML/automl_ijcnn15.pdf, which I think was mentioned in one of the other threads. I'll send an email to the authors, given that this has been discussed for a while now....